### ISSUE TITLE
Requesting Support for RK N80 Keyboard

### ISSUE THREAD

**Issue by Christoph-F** [rnayabed/rangoli#113]
Hello,

I've noticed that the RK N80 keyboard is not currently listed as supported in the project documentation. I would greatly appreciate it if support for the RK N80 could be added.

Thank you very much for considering my request.

**Comment by IvansZuks** [comment]
Yeah. I got this keyboard. Would be great to be able to configure it using my Linux laptop

**Comment by 10hertz** [comment]
I hope the developer can add support for this Keyboard on Mac OS.
